src/excalidraw-app/components/ExportToExcalidrawPlus.tsx

```diff
@@ -0,0 +1,99 @@
+      fill="currentColor"
+    />
+  </>,
+  { width: 89, height: 131, style: { transform: "translateX(4px)" } },
+);
+
+const encryptData = async (
+  key: string,
+  json: string,
+): Promise<{ blob: Blob; iv: Uint8Array }> => {
+  const importedKey = await getImportedKey(key, "encrypt");
+  const iv = createIV();
+  const encoded = new TextEncoder().encode(json);
+  const ciphertext = await window.crypto.subtle.encrypt(
+    {
+      name: "AES-GCM",
+      iv,
+    },
+    importedKey,
+    encoded,
+  );
+
+  return { blob: new Blob([new Uint8Array(ciphertext)]), iv };
+};
+
+export const ExportToExcalidrawPlus: React.FC<{
+  elements: readonly NonDeletedExcalidrawElement[];
+  appState: AppState;
+}> = ({ elements, appState }) => {
+  return (
+    <Card color="indigo">
+      <div className="Card-icon">{excalidrawPlusIcon}</div>
+      <h2>Excalidraw+</h2>
```

Comment:
i18n

Reply:
How do you wanna translate brand names?

Reply:
it will be `Excalidraw+` only but in different languages so it's readable in all languages is what I am thinking

Reply:
I don't think translating brand names makes sense, even to different alphabets.

Reply:
It might be not readable in other languages then, what I have seen is to make sure that translation doesn't go wrong in other languages for the brand name so `Original name (translated name)` is followed on the safer side. But since we are not sure so we can keep this for later.